src/renderer/src/@types/hooks.ts

```diff
@@ -9,6 +9,7 @@ export type TUseActionsActionState<T> = {
   isActing: boolean
   errors: TUseActionsErrors<T>
   changed: TUseActionsChanged<T>
+  isActed: boolean
```

Comment:
i don't think i understand this variable name. does it mean the it has already acted, so something like `hasActed`?